### PR TITLE
[WIP][Nidhi|Jaya] Add. fare category component

### PR DESCRIPTION
**Pull request by nidhirajput13**

### CODE REVIEW

File: src/components/Catalog.js [hunk 1]
```diff
@@ -13,25 +13,16 @@ const providerItems = (
   selectedItemId,
   selectedProviderId,
   selectedTravelClassId,
-  setSelectedTravelClassId,
+  onSelectTravelClass,
```
Comment (hariharanweb) [inline src/components/Catalog.js:16]: rename as onTravelClassSelect - guideline is on<Abc><Event>

File: src/components/Catalog.js [hunk 2]
```diff
@@ -53,12 +45,12 @@ const Catalog = ({
...
```
Comment (hariharanweb) [inline src/components/Catalog.js:48]: same here

**Review by hariharanweb — CHANGES_REQUESTED** [review]

File: src/components/Catalog.js (hunk 1 above)
Comment (nidhirajput13) [reply src/components/Catalog.js]: Fixed

File: src/components/Catalog.js (hunk 2 above)
Comment (nidhirajput13) [reply src/components/Catalog.js]: Fixed

**Pull request merged**